pkg/graph/resolvers/resolver.go

```diff
@@ -25,7 +25,8 @@ import (
 type Resolver struct {
 	store                *storage.Store
 	service              ResolverService
-	s3Client             *upload.S3Client
+	s3Client             *s3.S3Client
```

Comment:
Might want to flag this for a potential rename?
```suggestion
	s3Client             *s3.S3Client // TODO Maybe consider renaming to something like "fileUploadS3Client"
```

Reply:
I like it, I went ahead and renamed it. There might be some test configs that we find later we want to rename as well later